### DISCUSSION TITLE
Enhancement: tutorial pico mix

### DISCUSSION THREAD

**Discussion by Szymonkruk12** [FunkinCrew/Funkin#4020] [Ideas]
### Issue Checklist

- [X] I have properly named the enhancement
- [X] I have checked the issues/discussions pages to see if the enhancement has been previously suggested

### What is your suggestion, and why should it be implemented?

add it please

**Comment by AbnormalPoof** [comment]
This isn't the place to request new content...

**Comment by Hundrec** [comment]
Please close this issue and start a Discussion instead.
